hc/front/tests/test_add_pushbover.py

```diff
@@ -37,4 +38,14 @@ def test_it_validates_nonce(self):
         r = self.client.get("/integrations/add_pushover/?%s" % params)
         assert r.status_code == 403
 
-    ### Test that pushover validates priority
+    # Test that pushover validates priority
+    def test_that_pushover_validates_priority(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        params = "pushover_user_key=a&nonce=n&prio=7"
+        response = self.client.get(reverse("hc-add-pushover") + "?%s" % params)
```

Comment:
There's is a simpler and standard way of adding Params to the reverse function using **kwargs** and **args**
 here is an example of how to add kwargs.
https://stackoverflow.com/questions/13202385/django-reverse-with-arguments-and-keyword-arguments-not-found